src/components/Button/Button.Overview.stories.mdx

```diff
@@ -56,6 +57,11 @@ the color of the primary CTA but makes the hierarchy clear. The background is tr
       <Button variant="dark" isOutlined>
         Dark
       </Button>
+      <Box background="primary" padding="2xs">
```

Comment:
I added this background so the button is visible, if you all have a better solution for this please LMK!